addons/io_scene_gltf2/blender/exp/gltf2_blender_animate.py

```diff
@@ -1,637 +0,0 @@
-# Copyright 2018 The glTF-Blender-IO authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-#
-# Imports
-#
-
-import bpy
-from . import gltf2_blender_export_keys
-from . import gltf2_blender_extract
-from mathutils import Matrix, Quaternion, Euler
-
-
-#
-# Globals
-#
-
-JOINT_NODE = 'JOINT'
-
-NEEDS_CONVERSION = 'CONVERSION_NEEDED'
-CUBIC_INTERPOLATION = 'CUBICSPLINE'
-LINEAR_INTERPOLATION = 'LINEAR'
-STEP_INTERPOLATION = 'STEP'
-BEZIER_INTERPOLATION = 'BEZIER'
-CONSTANT_INTERPOLATION = 'CONSTANT'
```

Comment:
Do we still support CUBICSPLINE here? The code remaining seems much simpler than before, if that's still working as intended that's awesome. 👍

Reply:
It's mentioned multiple times in the new animation code so I do believe it.

Reply:
Great! Yes, tested and aside from one issue with custom spline tangents (https://github.com/KhronosGroup/glTF-Blender-IO/issues/140) it seems to be working as before. :)